GovUk.Frontend.Umbraco.ExampleApp/uSync/v9/Content/accordion.config

```diff
@@ -377,8 +377,5 @@
   ]
 }]]></Value>
     </blocks>
-    <headingScaleStart>
-      <Value><![CDATA[]]></Value>
-    </headingScaleStart>
   </Properties>
 </Content>
```

Comment:
This removal of `<headingScaleStart>` indicates you're missing a recent property, and as a result you've removed it from all pages.